### PR TITLE
test

### PR DESCRIPTION
**Pull request by CandiseS**
Hi Shirley,
Here's my assignment1.

### CODE REVIEW

File: app/src/utils.py [hunk 1]
```diff
@@ -7,3 +7,17 @@
 # for reference on exceptions, check the class notes here: https://github.com/FTEC-6v99/python-overview/blob/master/advanced/exceptions.py
 #
 # Make sure that you add type hints to the function paramter and return value
+
+import typing as t
+import math
```
Comment (Siyu-SG) [inline app/src/utils.py:12]: This can be omitted since we are just doing addition in the utils function.

**Review by Siyu-SG — COMMENTED** [review]
I have run through the unit tests, no error occurred. All worked fine!

File: app/tests/testgradecalc.py [hunk 2]
```diff
@@ -1,2 +1,15 @@
 # Use unittest library to create a unit test method to test the calculate_avg_grade function created in the gradecalc module
 # Make sure you include multiple unit tests to test that the function acts as expected in different scenarios
+import unittest
+from app.src.Student import Student
```
Comment (mab105120) [inline app/tests/testgradecalc.py:4]: Please add more tests to cover failures and edge cases.

File: app/tests/testutils.py [hunk 3]
```diff
@@ -1,2 +1,13 @@
 # Create a unit test method to test the calculate_avg function created in the utils module
 # Make sure you include multiple unit tests to test that the function acts as expected in different scenarios
+
+import unittest
+from app.src.utils import calc_avg
```
Comment (mab105120) [inline app/tests/testutils.py:5]: Please add more tests to cover failures and edge cases.

**Review by mab105120 — APPROVED** [review]
